builtin/credential/appgroup/path_tidy_user_id.go

```diff
@@ -0,0 +1,77 @@
+package appgroup
+
+import (
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathTidyUserID(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "tidy/user-id$",
+
```

Comment:
I removed safety buffer from this endpoint because the backend's periodic function would not allow any expired entries to live beyond 1 minute past its expiration. So, safety buffer did not make sense in this case to me, given that this 1 minute unavoidable buffer is also not ideal.

Unless, the periodic function should also refer to the configured safety buffer duration, which I did not think was necessary.